plumbing/cst/chain_state.go

```diff
@@ -50,6 +68,7 @@ func NewChainStateReadWriter(crw chainReadWriter, messages chain.MessageProvider
 	return &ChainStateReadWriter{
 		readWriter:      crw,
 		cst:             cst,
+		store:           newCarStore(cst),
```

Comment:
Create this on the fly rather than store it here - it's no actually new information nor expensive to create.